eio/tests/fuzz.ml

```diff
@@ -287,13 +309,17 @@ let main client_message server_message actions =
   ]
 
 let () =
-  Crowbar.(add_test ~name:"random ops" [bytes; bytes; list action] main)
-(*
-  Logs.(set_level (Some Info));
-  Logs.set_reporter (Logs_fmt.reporter ());
-  ignore action;
-  main "ping" "pong" [
-    Some (To_server, Send 5);
-    Some (To_client, Send 5);
-  ]
-*)
+  if true then
```

Comment:
sorry, I don't quite get this style of code? would you mind to explain the construct in more detail, and/or make it disappear. I've no clue in which cases the alternative should be executed (with manual code modification, likely).

Reply:
If you get a crash and you want to reproduce it manually for testing (e.g. to try variations) you can input it here directly. Keeping it in an `if` makes it easier to switch between the two modes, means it still gets type-checked when disabled, and you don't get bothered by unused variable errors from dune.

Reply:
Do you think it is possible to use "cmdliner" and have this "true" hidden behind a flag?